core/iwasm/interpreter/wasm.h

```diff
@@ -132,6 +144,11 @@ typedef void *table_elem_type_t;
 #if WASM_ENABLE_BULK_MEMORY != 0
 #define SECTION_TYPE_DATACOUNT 12
 #endif
+#if WASM_ENABLE_GC != 0
```

Comment:
Seems no need to add `#if ENABLE_GC != 0` here, it should be auto enabled when stringref is enabled. The below codes don't add the control like this.

Reply:
Got it~